src/pages/program.jsx

```diff
@@ -451,6 +451,16 @@ const Program = () => {
               title="Doors open"
               eventLocations={[locations.mainEnterence]}
             />
+            {isExtravaganza && (
+              <ProgramItem
+                dayStr="2022-09-16"
+                startTime="16:00"
```

Comment:
If I'm not mistaken, this will be up as soon as the doors open. Is it?

Reply:
I assumed it opens when doors open, but actually still waiting for Griffin's response